<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adduserdialog</class>
 <widget class="QDialog" name="adduserdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>439</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>439</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>adduser</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>321</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>用户名： </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtMaterialAutoComplete" name="L_Name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>账   号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtMaterialAutoComplete" name="L_Account"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>密   码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtMaterialAutoComplete" name="L_pswd"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>性别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Rd_Male">
        <property name="text">
         <string>男</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Rd_Female">
        <property name="text">
         <string>女</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QtMaterialRaisedButton" name="BtnReg">
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QtMaterialRaisedButton" name="BtnBack">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialAutoComplete</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialautocomplete.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
